README.md

```diff
@@ -115,10 +115,16 @@ This will allow you to run challenge 1-8.
 
 ### AWS challenge 9, 10 and 11
 
-*Can be used for challenges 1-11*
+_Can be used for challenges 1-11_
 
 Follow the steps in [the README in the AWS subfolder](aws/README.md).
 
+### GCP challenge 9 and 10
+
+_Can be used for challenges 1-10_
+
+Follow the steps in [the README in the GCP subfolder](gcp/README.md).
```

Comment:
please add that this is experimental and not done yet (refer to the issues in GH for now ;D )

Reply:
Added 'Experimental' but didn't add the issues - can we do without the links?

Reply:
```suggestion
Follow the steps in [the README in the GCP subfolder](gcp/README.md). Note that this is [still](https://github.com/commjoen/wrongsecrets/issues/39) [work in progress](https://github.com/commjoen/wrongsecrets/issues/40)
```